README.md

```diff
@@ -2,7 +2,7 @@ libgit2 - the Git linkable library
 ==================================
 
 [![Travis Build Status](https://secure.travis-ci.org/libgit2/libgit2.svg?branch=master)](http://travis-ci.org/libgit2/libgit2)
-[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/gnjsdi9r48cfoveg/branch/master?svg=true)](https://ci.appveyor.com/project/nulltoken/libgit2/branch/master)
+[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/gnjsdi9r48cfoveg/branch/master?svg=true)](https://ci.appveyor.com/project/libgit2/libgit2/branch/master)
```

Comment:
`https://ci.appveyor.com/api/projects/status/gnjsdi9r48cfoveg/branch/master?svg=true` generates the badge for nulltoken/libgit2

You should find the proper badge url in the Settings tab